.github/compatibility-test-matrices/release-v2.5.x/transfer.json

```diff
@@ -0,0 +1,15 @@
+{
+  "chain-a": ["release-v2.5.x", "v2.4.2"],
+  "chain-b": ["release-v2.5.x", "v2.4.2"],
+  "entrypoint": ["TestTransferTestSuite"],
+  "test": [
+    "TestMsgTransfer_Succeeds_Nonincentivized",
+    "TestMsgTransfer_Fails_InvalidAddress",
+    "TestMsgTransfer_Timeout_Nonincentivized",
+    "TestMsgTransfer_WithMemo",
+    "TestSendEnabledParam",
+    "TestReceiveEnabledParam"
+  ],
```

Comment:
It might be nice if in the future we can just have test exclusions, I forget why it was difficult to list all possible tests